src/test/java/taskFinal/util/SingleWebDriver.java

```diff
@@ -0,0 +1,173 @@
+package taskFinal.util;
+
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+import static taskFinal.util.TestConfig.*;
+
+public final class SingleWebDriver {
+    private static SingleWebDriver instance;
+    private WebDriver driver;
+    private SingleWebDriver() {}
+
+    public WebDriver getTargetWebDriver() throws MalformedURLException {
+        switch (TARGET)
+        {
+            case "Local":
```

Comment:
It is not the issue, but it is better to have options for switch case in upper case

Reply:
Done